js/ui/save-html.js

```diff
@@ -75,6 +80,17 @@ define(
                 }
                 ui.freshModal("Save Snapshot", $div);
             }
+        // Create and download an EPUB 3 version of the content
+        // Using (by default) the EPUB 3 conversion service set up at labs.w3.org/epub-generator
+        // For more details on that service, see https://github.com/iherman/respec2epub
+        ,   makeEPubHref: function(){
+                if (!conf.epubGenTool) {
+                    conf.epubGenTool = "https://labs.w3.org/epub-generator/cgi-bin/epub-generator.py";
+                }
+                var epubHref = conf.epubGenTool;
+                epubHref+= "?type=respec&url=" + doc.location.href;
```

Comment:
> On 08 Sep 2015, at 10:49 , Tobie Langel <notifications@github.com> wrote:
> 
> In js/ui/save-html.js <https://github.com/w3c/respec/pull/487#discussion_r38902594>:
> 
> > @@ -75,6 +80,17 @@ define(
> >                  }
> >                  ui.freshModal("Save Snapshot", $div);
> >              }
> > +        // Create and download an EPUB 3 version of the content
> > +        // Using (by default) the EPUB 3 conversion service set up at labs.w3.org/epub-generator
> > +        // For more details on that service, see https://github.com/iherman/respec2epub
> > +        ,   makeEPubHref: function(){
> > +                if (!conf.epubGenTool) {
> > +                    conf.epubGenTool = "https://labs.w3.org/epub-generator/cgi-bin/epub-generator.py";
> > +                }
> > +                var epubHref = conf.epubGenTool;
> > +                epubHref+= "?type=respec&url=" + doc.location.href;
> That's unrelated to Respec, really. This is just to make sure you URL is valid.
> 
> 

But isn't the URL valid already if I get that far? After all, this thing comes in at the end of the respec processing, so to say...